examples/platform/silabs/SiWx917/SiWx917/sl_wlan_config.h

```diff
@@ -28,7 +28,16 @@
 //! Disable feature
 #define RSI_DISABLE 0
 
-#define SI91X_LISTEN_INTERVAL 0
+#define IVT_OFFSET_ADDR 0x8202000         /*<!Application IVT location VTOR offset>        */
```

Comment:
is this file the right place for those defines? It isn't related to wlan configs

Reply:
Yeah, removed these defines. Already these defines are added in the matter support